FORMAT_JSON.md

```diff
@@ -0,0 +1,121 @@
+# Implementation Details: JSON
+
+## General Notes
+
+JSON is a format build on attribute-value pairs (dictionaries) and arrays.
+Its most common representation is as strings.
```

Comment:
I don't quite understand this one. Do you mean that JSON is not a binary format, but a text file?